README.md

```diff
@@ -0,0 +1,77 @@
+# React Compass
+
+This project provides a set of components for the React Compass Design System that can be used to
+implement user experiences consistent with
+[ComfortDelGro's design principles](https://comfortdelgro.github.io/compass-design).
+
+<a href="./LICENSE">
+  <img src="https://img.shields.io/badge/license-MIT-blue.svg" alt="CofortDelGro's React Compass is released under the MIT license" />
+</a>
+<a href="https://turbo.build/">
+  <img src="https://img.shields.io/badge/maintained%20with-turbo-cc00ff.svg" alt="Maintained with Turborepo" />
+</a>
+<a href="https://github.com/comfortdelgro/compass-design/actions/workflows/release.yml">
+  <img alt="Release" src="https://github.com/comfortdelgro/compass-design/actions/workflows/release.yml/badge.svg">
+</a>
+---
+
+## Getting started
+
+### React
+
+**Installation**
+
+To get started using Canvas kit React first add or install the module to your existing React project
+
+```sh
+yarn add @comfortdelgro/react-compass
+```
+
+or
+
+```sh
+npm install @comfortdelgro/react-compass
+```
+
+or
+
+```sh
+pnpm install @comfortdelgro/react-compass
+```
+
+**Usage**
+
+```tsx
+import {Alert} from '@comfortdelgro/react-compass'
```

Comment:
Let's also showcase another way which is
```tsx
import Alert from '@comfortdelgro/react-compass/Alert'
```
so that the users know we support [tree-shaking](https://developer.mozilla.org/en-US/docs/Glossary/Tree_shaking).

Reply:
Done